#24749 updated netlify plugin doc (#254)
flotiq/flotiq-docs: docs/panel/Plugins/Netlify-integration.md

## Installing the Netlify plugin

Find in the right sidebar panel `Plugins` and select

![Flotiq plugins](images/sidebar-plugins.png){: .center .width25 .border}

> On the next screen, click on the plus next to `Netlify Build` for adding plugin.

Reviewer: ```suggestion
On the next screen, click the plus next to `Netlify Build` to add a plugin.
```
style fix

![Adding Netlify Build plugin to Flotiq](images/netlify/plugins-screen-netlify.png){: .center .width75 .border}

> Once plugin is enabled, click `Mange` to open modal with the form.

Reviewer: ```suggestion
Once the plugin is enabled, click `Manage` to open the modal with the form.
```
typos
